src/loop/typer.ml

```diff
@@ -989,9 +1013,21 @@ module Make(S : State_intf.Typer with type ty_state := ty_state) = struct
       S.error ~loc st unbound_type_wildcards (env, tys)
     | T.Unhandled_ast ->
       S.error ~loc st unhandled_ast (env, fragment)
+    (* Alt-Ergo Functional Array errors *)
+    | Ae_Arrays.Bad_farray_arity ->
+      S.error ~loc st bad_farray_arity ""
+    (* Alt-Ergo Arithmetic errors *)
+    | Ae_Arith.Expected_arith_type ty ->
+      S.error ~loc st expected_arith_type (ty, "")
+    | Ae_Arith.Incompatible_Arith_Types (tya, tyb) ->
+        S.error ~loc st incompatible_arith_types
+        ((tya, tyb), "Alt-Ergo's native language is strongly typed. Only the real \
```

Comment:
This comment about "the real exponentiation operator" is confusing, particularly since the operator can operate either on ints or reals. The current wording makes it seem as if the exponentiation always return a real (while taking either ints or reals).

Reply:
Alt-Ergo's native language has two exponentiation operators ```**``` and ```**.```. ```**``` takes only ints and returns ints while ```**.``` can take either ints or reals or both and returns a real. In the comment I was referring to ```**.```.